common/services/user.test.js

```diff
@@ -1,3 +1,5 @@
+const axios = require('axios')
+const { expect } = require('chai')
```

Comment:
I don't think this needs to be imported as it's already defined in the test globals